src/main/scala/uk/gov/nationalarchives/tdr/api/model/file/NodeType.scala

```diff
@@ -15,14 +15,14 @@ object NodeType {
       if (deconstructed.length == 1) None else Some(deconstructed.dropRight(1).mkString("/"))
 
     def fileType: String = {
```

Comment:
Maybe rename this to: `typeIdentifier` to make it more consistent, as not strictly a "file type" anymore?

Reply:
Makes sense, I've changed it.